### ISSUE TITLE
Jenkins builds appearing as stuck

### ISSUE THREAD

**Issue by aantron** [ocsigen/lwt#475]
Looking at [recent PRs](https://github.com/ocsigen/lwt/pulls), the Jenkins status is pending on the last three (#474, #471, #469), and has been so for weeks. @vasilisp could this be related to the recent infrastructure changes that the Ocsigen website also suffered from?

**Comment by aantron** [comment]
In addition, it looks like Jenkins is no longer running on `master`, and therefore not updating the ocsigen.org manual. For example, compare this commit https://github.com/ocsigen/lwt/commit/af91fad3c07223bf11e917daa2bbd3b283acdcad, and the dev manual at https://ocsigen.org/lwt/dev/api/Lwt_log. Looking at ocisgen.org-data, I see the [last commit](https://github.com/ocsigen/ocsigen.org-data/commits/master) was on July 27.

**Comment by aantron** [comment]
I disabled the `ci.inria.fr` webhooks for now. Not sure if that is sufficient to stop the stuck Jenkins status from appearing, but I hope so.

They webhooks are still there, so they can be easily re-enabled whenever the Jenkins setup is getting fixed/debugged.

**Comment by aantron** [comment]
Closing this in favor of tracking elsewhere, e.g. https://github.com/ocsigen/ocsigen.org-data/issues/13.
